### DISCUSSION TITLE
Validating the purchase & subscription on the backend side

### DISCUSSION THREAD

**Discussion by darimuhittinhey** [hyochan/react-native-iap#2589] [Q&A]
I come across the term "validation of the purchase by the backend" in many places. What does this mean ? And how to do it ? For example, what should be used for a backend using nodejs ? What data should we store - use from the values returned to us by react-native-iap for the validation process ?

I need all kinds of ideas you can give. Even if it is not among my questions.

Thank you in advance.

**Comment by muhammadsaadullah1** [comment]
once the purchase is done you run a function that could call an api  for examaple , 
this is the product purchased , and this user purchased  it  and store it in backend , it can be of any kind.
Like if user does subscribe  you call a function to set user as subscribed = true in your prefered Database.
